src/planscape/forsys/get_forsys_inputs_test.py

```diff
@@ -219,19 +204,25 @@ def test_gets_forsys_input_ignoring_nan(self):
         headers = ForsysInputHeaders(params.priorities)
 
         input = ForsysGenerationInput(params, headers)
-        _assert_dict_almost_equal(self, input.forsys_input, {
-            'proj_id': [0, 0, 0, 0],
-            'stand_id': [0, 1, 2, 3],
-            'area': [.09, .09, .09, .09],
-            'cost': [450000000, 450000000, 450000000, 450000000],
-            'p_foo': [.95, .94, .93, .92],
-            'p_baz': [.8, .8, .8, .8],
-            'c_foo': [.95, .94, .93, .92],
-            'c_baz': [.8, .8, .8, .8],
+        assert_dict_almost_equal(self, input.forsys_input, {
+            'proj_id': [0, 0, 0, 0, 0, 0, 0, 0],
+            'stand_id': [0, 1, 2, 3, 4, 5, 6, 7],
+            'area': [.09, .09, .09, .09, .09, .09, .09, .09],
+            'cost': [450000000, 450000000, 450000000, 450000000,
+                     450000000, 450000000, 450000000, 450000000],
+            'p_foo': [.95, .94, .93, .92, 0, 0, 0, 0],
+            'p_baz': [.8, .8, .8, .8, 0, 0, 0, 0],
+            'c_foo': [.95, .94, .93, .92, 0, 0, 0, 0],
+            'c_baz': [.8, .8, .8, .8, 0, 0, 0, 0],
```

Comment:
Kind of: the condition and priority values of any stands that aren't eligible for treatment are set to 0.

So for a stand where foo is nan and baz is 0.6: because not all conditions are available, the stand isn't eligible for treatment. Because the stand isn't eligible for treatment, its priority and condition values are set to 0.

I should add a comment for this since it's an arbitrary ruling made by Planscape.

Reply:
Done.

Reply:
@strangeties I think this is the logic I am confused about. Why are stands that are missing a score set to "not treatable"? My understanding from conversations with Nick was that missing values would be filled in with a default value, e.g. 0 or -1, provided by USFS. They may very well still have a high overall weighted condition score and therefore be included in a project.

Reply:
I see.

That's a different topic.

Values may be missing before a raster is uploaded. These values should be set to a default suggested by experts before being uploaded to postGIS (or EarthEngine moving forward).

This PR focuses on stands that are ineligible for treatment but could still be included in projects. This would encompass 1) roads 2) buildings 3) pixels that just happen to be missing condition values at retrieval time. The impact score in this situation should be 0.

Reply:
please note we're still using normalized condition rasters so case 3) is much more prevalent than they should be for AP scores.